abnf/odata-abnf-construction-rules.txt

```diff
@@ -315,7 +315,7 @@ orderbyItem = commonExpr [ RWS ( "asc" / "desc" ) ]
 skip = ( "$skip" / "skip" ) EQ 1*DIGIT
 top  = ( "$top"  / "top"  ) EQ 1*DIGIT
 
-index  = ( "$index" / "index" ) EQ 1*DIGIT
+index  = ( "$index" / "index" ) EQ *SIGN 1*DIGIT

```

Comment:
What about constraining the domain in the grammar via `[  MINUS ]` (Or the literal which the github webui on ios does not let me enter) instead of `*SIGN`. That makes life for implementers easier and is maybe more close to our intent. Maybe we do not want to support index expressions like `—+——42` :wink:

Reply:
@sthagen Thanks for catching this!